baseplate/observers/prom_metrics_tagged.py

```diff
@@ -0,0 +1,198 @@
+from random import random
+from time import time
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from prometheus_client import Counter
+from prometheus_client import Histogram
+
+from baseplate import _ExcInfo
+from baseplate import BaseplateObserver
+from baseplate import LocalSpan
+from baseplate import RequestContext
+from baseplate import Span
+from baseplate import SpanObserver
+from baseplate.lib import config
+
+
+class _PrometheusMetrics:
+    def __init__(self, allowlist: List[str]):
+        self.server_rate = Counter(
+            "baseplate_server_span_rate", "Baseplate Server Span Counter", allowlist
+        )
+        self.server_latency = Histogram(
+            "baseplate_server_span_latency_seconds", "Baseplate Server Span Latency", allowlist
+        )
+        self.client_rate = Counter(
+            "baseplate_client_span_rate", "Baseplate Client Span Counter", allowlist
```

Comment:
Assuming the expectation is that if a client want another metric that it's implemented directly

Reply:
They can set (and allow) a tag on the span, and it will get shipped. It's much nicer for devs to just make a new metric.